src/styles/main.scss

```diff
@@ -0,0 +1,163 @@
+@import 'variables';
+@import 'stars';
+@import 'header';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.products {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+
+.products__wrapper {
+  margin: 0 auto;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: $column-size;
+  grid-template-rows: auto;
+  justify-items: center;
+  justify-content: center;
+  grid-gap: 46px 48px;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $column-size);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, $column-size);
+  }
+
+  @media (min-width: 488) and (max-width: 767) {
```

Comment:
You missed the 'px' unit in your media query. It should be `@media (min-width: 488px) and (max-width: 767px)`.